app/views/users/_organization_user.html.erb

```diff
@@ -2,7 +2,7 @@
   <td><%= user.name %></td>
   <td><%= user.email %></td>
   <td><%= user.kind %> </td>
-  <td><%= user.most_recent_sign_in %></td>
+  <td><%= user.current_sign_in_at %></td>
```

Comment:
I'm having a little trouble following these changes, can you help me know why they're necessary?